kimchi/src/circuits/polynomials/rot.rs

```diff
@@ -198,7 +201,7 @@ where
     // (stored in coefficient as a power-of-two form)
     //   * Operates on Curr row
     //   * Shifts the words by `rot` bits and then adds the excess to obtain the rotated word.
-    fn constraint_checks<T: ExprOps<F>>(env: &ArgumentEnv<F, T>) -> Vec<T> {
+    fn constraint_checks<T: ExprOps<F>>(env: &ArgumentEnv<F, T>, _cache: &mut Cache) -> Vec<T> {
```

Comment:
What's the reason to expose an unused parameter?

Reply:
I suspect it's because this is implemented as a trait. Can you confirm this?

Reply:
That's correct, yes. The function signature is fixed by the trait.